File: 백준/Bronze/2693. N번째 큰 수/N번째 큰 수.cc
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
    int testCase;
    int inputNumber;
    int inputArray[10];

    cin >> testCase;

    for (int i = 0; i < testCase; i++) {
        for (int j = 0; j < 10; j++) {
            cin >> inputNumber;

            inputArray[j] = inputNumber;
        }

        sort(inputArray, inputArray + 10);

        cout << inputArray[7] << " ";
    }

    return 0;
}

File: 백준/Bronze/2693. N번째 큰 수/N번째 큰 수.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s1 = new Scanner(System.in);
        int testCase;
        int inputNumber;
        int[] inputArray = new int[10];

        testCase = s1.nextInt();

        for (int i = 0; i < testCase; i++) {
            for (int j = 0; j < 10; j++) {
                inputNumber = s1.nextInt();

                inputArray[j] = inputNumber;
            }

            Arrays.sort(inputArray);

            System.out.println(inputArray[7]);
        }

        s1.close();
    }
}
